libperiph/vendor/arm/cortexm/stm32/f4/src/stm32f4xx_ll_rcc.c

```diff
@@ -765,7 +765,7 @@ uint32_t LL_RCC_GetSDIOClockFreq(uint32_t SDIOxSource)
   * @retval RNG clock frequency (in Hz)
   *         - @ref  LL_RCC_PERIPH_FREQUENCY_NO indicates that oscillator is not ready
   */
-uint32_t LL_RCC_GetRNGClockFreq(uint32_t RNGxSource)
+uint32_t LL_RCC_GetRNGClockFreq(uint32_t __attribute__((unused)) RNGxSource)
```

Comment:
Why does RNGxSource have to be unused?

Reply:
for some microcontrollers that are selected by the ifdef directive the parameter is not used which generates a warning